mobile/components/AppDrawer.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import Drawer from 'react-native-drawer';
+import { Actions, DefaultRenderer } from 'react-native-router-flux';
+import PropTypes from 'prop-types';
+
+import Menu from './Menu';
+
+const drawerStyles = {
+  drawer: {
+    shadowColor: '#000',
+    shadowOpacity: 0.3,
+    shadowRadius: 1,
+  },
+  main: {
+    paddingLeft: 0,
+  },
+};
+
+class AppDrawer extends Component {
+  state = {};
+
+  render() {
+    const state = this.props.navigationState;
+    const { ...children } = state.children;
+    return (
+      <Drawer
+        open={state.open}
+        onOpen={() => Actions.refresh({ key: state.key, open: true })}
```

Comment:
You should be aware of putting functions inline in component params. Better, if possible, please make binding as there is in Collapsible Component ex. "this.toggle = this.toggle.bind(this);".

The reason is performance, and sometimes it can be not nice.

Reply:
https://github.com/aksonov/react-native-router-flux/blob/v4-docs/docs/OTHER_INFO.md#drawer-side-menu-integration

Reply:
Ok, let's leave it.